.code-samples.meilisearch.yaml

```diff
@@ -454,3 +454,18 @@ security_guide_list_keys_1: |-
 security_guide_delete_key_1: |-
   client = MeiliSearch::Client.new('http://127.0.0.1:7700', 'masterKey')
   client.delete_key('d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4')
+tenant_token_guide_generate_sdk_1: |-
+  api_key = 'B5KdX2MY2jV6EXfUs6scSfmC'
```

Comment:
Why the apiKey is so short? I'm not sure is possible that a key generated by `Meilisearch` looks like that.
I will prefer that the key should look like a real key like the one from line 456.

Reply:
I'm not convinced by the long one. What about: 
```ruby
api_key = 'B5KdX2MY2jV6EXfUs6scSfmC...'
```

?

Reply:
Is not the fact that the key is short but more that I think to have examples that look like real data is better no?

Reply:
I like the idea of adding the `...` :D (I thought about using a long and real API key, but I didn't like it because it wasn't so helpful in the context of the code sample).